sway-book/src/documentation/language/variables/index.md

```diff
@@ -0,0 +1,14 @@
+# Variables
+
+A variable is a way to reference some information by a specific name and it can take the form of a variety of [data structures](../built-ins/index.md).
+
+In Sway there are two states that a variable can take:
+
+- immutable
+  - Can be read but cannot be changed after it has been declared
```

Comment:
I would move this to the previous line:
" - Immutable: can be read but cannot be changed after it has been declared"

Reply:
I don't understand